### DISCUSSION TITLE
Incomplete result for a perfectly formed address

### DISCUSSION THREAD

**Discussion by pedrocamargo** [mediagis/nominatim-docker#536] [General]
I have deployed the nominatim docker on my local server with data covering only the US (configuration below).

When running a few searches, however, the results from the OSM nominatim server are so different that it suggests there is missing data on my local server. The example address '**1724 9th Avenue North, Bessemer, AL 35020, United States**', for example:

My local service returns

**Location(9th Avenue North, Bessemer, Jefferson County, Alabama, 35020, United States, (33.4092612, -86.9587683, 0.0))**

While the OSM nominatim server returns

**Location(1724, 9th Avenue North, Bessemer, Jefferson County, Alabama, 35020, United States, (33.40477053061225, -86.96224195918367, 0.0))**

It is clear that my local service did not find the house/building number, and that error persist even when I change the number to others known to exist.

Any insights would be GREATLY appreciated!

![image](https://github.com/mediagis/nominatim-docker/assets/7377666/97a77a71-b424-4f40-b4f9-3a61814fa83b)

**Comment by pedrocamargo** [comment]
I forgot to mention that this also happens for many other streets in the same city, which leads me to believe that I failed to do some key configuration of the server

**Comment by mtmail** [comment]
That address is part of the optional TIGER dataset. When you click 'details' on https://nominatim.openstreetmap.org/ui/search.html?q=1724+9th+Avenue+North%2C+Bessemer%2C+AL+35020%2C+United+States it will say 'not found', that's a sign it didn't come from OSM data.

I see you already specified TIGER data to be imported so it should've already worked. It's possible to install it later, it takes a couple of hours, see https://nominatim.org/release-docs/latest/customize/Tiger/

**Reply by pedrocamargo** [reply]
I would have never found that by myself!  Thank you!!!

I do end up with an error after that, though

nominatim add-data --tiger-data tiger-nominatim-preprocessed-latest.csv.tar.gz
2024-03-05 21:38:01: Using project directory: /nominatim
2024-03-05 21:38:01: FATAL: Cannot connect to database: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: FATAL:  role "root" does not exist

Maybe that's why the import of the tiger data failed in the first place?

**Reply by mtmail** [reply]
It usually works as part of the installation, maybe the data file download failed.

Try `sudo -E -u nominatim nominatim add-data ...`

**Reply by pedrocamargo** [reply]
NAILED!  Thanks!
